src/sinks/gcp_chronicle/chronicle_unstructured.rs

```diff
@@ -52,6 +52,35 @@ use crate::{
     tls::{TlsConfig, TlsSettings},
 };
 
+
+/// Compression configuration.
+#[configurable_component]
+#[derive(Clone, Debug, Derivative, Eq, PartialEq)]
```

Comment:
nit: If you add `Copy` to the list of derives, you can avoid an unnecessary `clone` below.